### PR TITLE
Test js

### PR DESCRIPTION
**Pull request by Khanetmoi**
mobile menu with js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,39 +2,59 @@
...
+                    <h3>Contact</h3>
+                    <a href=#><i class="fa-solid fa-angle-right"></i></a>
+                </div>
+            </li>
+            <li>
+                <div class="navstyle headvisible">
+                    <h3>About</h3>
+                    <a href=#><i class="fa-solid fa-angle-right"></i></a>
+                </div>
+            </li>
```
Comment (thecodechaser) [inline index.html]: - [x] Good job on the functionality, but there is a little issue. When the user clicks any of the mobile menu options, the mobile menu should disappear and a correct part of the page should be displayed.
But this functionality is not working. When I click on the menu options the mobile menu is still there. Please make sure when the user clicks any of the mobile menu options, the mobile menu disappears and a correct part of the page should be displayed
 - You may want to add events to your mobile menu options so when any of the event is fired you can use `display: none` for mobile menu

Comment (thecodechaser) [inline index.html]: - [x] When I open the mobile menu and switch to the desktop screen the mobile menu is still on the screen it should disappear. Please make sure when we switch the screen from mobile to desktop the mobile menu should disappear.
 - You can use media queries for that if the screen cross the min-width you can set your mobile menu `display: none`

### See the image for reference:

image

Comment (thecodechaser) [inline index.html]: - [x]  Good job the design, But you are supposed to stick to the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design as much as possible. Your design is a little different from the template. Please use correct font size, weight, and correct color for your buttons and make sure it matches the design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Khanetmoi👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Khanetmoi, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**